packages/gatsby-remark-autolink-headers/README.md

```diff
@@ -27,6 +27,7 @@ module.exports = {
 ## Options
 
 - `offsetY`: Signed integer, vertical offset value in pixels, e.g.
+- `icon`: Use your own svg icon
```

Comment:
Maybe worth mentioning that both options are optional? Other than maybe this, this PR is ready to go in

Reply:
Did update the README 👍